### PR TITLE
29 admin merchant create

### LINKED ISSUE #18: 29. Admin Merchant Create
As an admin,
When I visit the admin merchants index (/admin/merchants)
I see a link to create a new merchant.
When I click on the link,
I am taken to a form that allows me to add merchant information.
When I fill out the form I click ‘Submit’
Then I am taken back to the admin merchants index page
And I see the merchant I just created displayed
And I see my merchant was created with a default status of disabled.

### PR DESCRIPTION
**Pull request by garrettgregor**
closes #18 

# Refactor Opportunities
- removing `<br>`'s from form separating nav bar - lots of them now

# Test Coverage
✅  100%* (`ber` is now getting more conflicts with test data having duplicate results)
Screenshot 2023-06-06 at 7 09 17 PM

## Notes
Renamed `edit_spec` back to `update_spec` - @JCBeldo looking back at adopt-dont-shop repo had named spec for edit as `update_spec.rb`:
Screenshot 2023-06-06 at 7 07 24 PM
Screenshot 2023-06-06 at 7 07 32 PM

### CODE REVIEW

File: app/views/admin/merchants/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<%= render partial: "admin/admin_nav" %>
+<br>
+<br>
+<br>
+<br>
+<br>
+<br>
+<%# Refactor these br's later to format padding/argins better  %>
+<%= form_with model: Merchant.new, url: admin_merchants_path do |f| %>
```
Comment (JCBeldo) [inline app/views/admin/merchants/new.html.erb:9]: Hey, remember to call this in the controller and then just cal the variable in the view.

Comment (JCBeldo) [reply app/views/admin/merchants/new.html.erb:9]: Sorry, Merchant.new in the controller and @merchant in the view

File: db/seeds.rb [hunk 2]
```diff
@@ -12,12 +12,3 @@
 Invoice.destroy_all
 Merchant.destroy_all
 Customer.destroy_all
-
-
```
Comment (JCBeldo) [inline db/seeds.rb:16]: deleting these to avoid the seed loading problems?

**Review by JCBeldo — APPROVED** [review]
Looks good, but might want to update the model to avoid MVC infractions.

File: app/views/admin/merchants/new.html.erb (hunk 1 above)
Comment (garrettgregor) [reply app/views/admin/merchants/new.html.erb:9]: Thanks for the reminder! I'll update this in my current branch since this already got merged

File: db/seeds.rb (hunk 2 above)
Comment (garrettgregor) [reply db/seeds.rb:16]: yes
